Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml

```diff
@@ -0,0 +1,2251 @@
+          description: 'The type of the workspace.'
+          type: String
+        - contextPath: Bitbucket.Project.workspace.uuid
+          description: 'The project workspace universal unique id.'
+          type: String
+        - contextPath: Bitbucket.Project.workspace.name
+          description: 'The name of the project workspace.'
+          type: String
+        - contextPath: Bitbucket.Project.workspace.slug
+          description: 'The slug of the project workspace.'
+          type: String
+        - contextPath: Bitbucket.Project.workspace.links.avatar.href
+          description: 'A link to the project workspace photo.'
+          type: String
+        - contextPath: Bitbucket.Project.workspace.links.html.href
+          description: 'A link to the project workspace repositories.'
+          type: String
+        - contextPath: Bitbucket.Project.workspace.links.self.href
+          description: 'A link to the result json.'
+          type: String
+        - contextPath: Bitbucket.Project.key
+          description: 'The project key.'
+          type: String
+        - contextPath: Bitbucket.Project.uuid
+          description: 'the project universal unique id.'
+          type: String
+        - contextPath: Bitbucket.Project.is_private
+          description: 'Is the project private or not.'
+          type: Boolean
+        - contextPath: Bitbucket.Project.name
+          description: 'The project name.'
+          type: String
+        - contextPath: Bitbucket.Project.description
+          description: 'The project description.'
+          type: String
+        - contextPath: Bitbucket.Project.links.self.href
+          description: 'A link to the result json.'
+          type: String
+        - contextPath: Bitbucket.Project.links.html.href
+          description: 'A link to the project repositories.'
+          type: String
+        - contextPath: Bitbucket.Project.links.repositories.href
+          description: 'A link to the project repositories.'
+          type: String
+        - contextPath: Bitbucket.Project.links.avatar.href
+          description: 'A link to the project photo.'
+          type: String
+        - contextPath: Bitbucket.Project.created_on
+          description: 'The date the project was created.'
+          type: String
+        - contextPath: Bitbucket.Project.updated_on
+          description: 'The date the project was updated.'
+          type: String
+        - contextPath: Bitbucket.Project.has_publicly_visible_repos
+          description: 'Does the project has a publicly visible repositories or not.'
+          type: Boolean
+    - name: bitbucket-open-branch-list
+      arguments:
+        - name: repo
+          required: false
+          description: the repository name or slug.
+          isArray: false
+        - name: limit
+          required: false
+          isArray: false
+          description: The maximum number of items in the list.
```

Comment:
To all limit and page arguments, add the default value to the descriptions. 
`The default value is 50`